src/App.tsx

```diff
@@ -1,17 +1,28 @@
-import React from 'react';
+import React, { useState, useMemo } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const filteredMovies = [...moviesFromServer];
+  const [query, getQuery] = useState('');
+
+  const visibleMovies: Movie[] = useMemo(
+    () => filteredMovies.filter((movie) => {
+      return movie.title.toLowerCase().includes(query.toLowerCase())
+      || movie.description.toLowerCase().includes(query.toLowerCase());
+    }),
+    [filteredMovies],
```

Comment:
Add 'query' to the useMemo dependencies array, so that it recalculates visibleMovies when query changes.